libs/domain/content/src/services/adapter/content-experience.adapter.spec.ts

```diff
@@ -13,6 +13,7 @@ describe('ContentExperienceAdapter', () => {
   let contentService: ContentService;
 
   beforeEach(() => {
+    mockFeatureVersion('1.4');
```

Comment:
Don't you need to test < 1.4 as well?

Reply:
I think it's will be redundant tests, cause it's a bit artificial versioning

Reply:
The idea is to be sure that the < 1.4 functionality is still working as was

Reply:
Ideally we would test different versions, true. We're not doing it often though.